### PR TITLE
Bug 1254052 - [TV][Home] Folders should show small icons for its containing cards

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c5e12b0f11d9309a675d08a3fc4ac8002fa246ac

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=46d03950643731c4f4a364db0b02a229ee9da434

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=70fda041e5d4834292953242514da45cc512b34d

File: tv_apps/smart-home/js/card_util.js [hunk 1]
```diff
@@ -162,7 +213,9 @@
         this._fillCardIcon(cardButton, card);
       } else if (card instanceof Folder) {
         cardButton.setAttribute('app-type', 'folder');
-        cardButton.dataset.icon = 'folder';
+        this._updateFolderCardIcons(cardButton, card);
```
Comment (rexboy7) [inline tv_apps/smart-home/js/card_util.js]: May you try put this to `home.onCardInserted`?

File: tv_apps/smart-home/js/card_util.js [hunk 2]
```diff
@@ -162,7 +213,9 @@
...
+        card.on('folder-changed',
```
Comment (rexboy7) [inline tv_apps/smart-home/js/card_util.js]: May you try insert listener at `home.onCardInserted`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=249285feb04cc73e221ba61a2e2709f38c7bee95

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ed51cf86d7c1106a7efb06f05e98ac69f829fe42

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3c69d1a5021cc9b54296b61c5fb07c8673d280c1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=24ec00be07a315ba9e440094e3204e627a8a953c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc465d1f7b29d124634dd602eb4e2d59a3c5e597

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9c3d0e474a0bee85a87efd20fd1fdfae369a68e1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d347bbbdd0e78498f268989d9d1d6c307f19867a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4f5c29905cb5d88de02fc5cba8e3c27d8ed2c865

**Pull request merged**